harvey4/src/main/kotlin/factory/factory/PiazzaFactory.kt

```diff
@@ -0,0 +1,9 @@
+package factory.factory
+
+import factory.pizza.Pizza
+
+interface PiazzaFactory {
+    fun createCheesePizza(): Pizza
```

Comment:
```suggestion
    fun createPizza(pizzaType): Pizza
```
으로 하는게 어떨까요? 피자가 cheese인지 potato인지 pizza factory가 모르는게 더 좋지 않을까요?

Reply:
좋은 생각입니다